### PR TITLE
Some readme updates!

### PR DESCRIPTION
**Pull request by montehurd**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
-# applib
-Cross-platform JavaScript library for Wikimedia apps
+# Library for common Javascript transforms used by both the Android and iOS Wikipedia apps
```
Comment (niedzielski) [inline readme.md]: Would you mind keeping the project title as the highest level element?

```markdown
# applib
Library for common Javascript transforms used by both the Android and iOS Wikipedia apps
```

Also consider rephrasing to include CSS: `..Javascript transforms and CSS...`.

Comment (niedzielski) [inline readme.md]: I believe the "s" in [JavaScript](https://en.wikipedia.org/wiki/JavaScript) is always capitalized. Please correct me if I'm mistaken!

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+## What Applib is for
+* Javascript transforms common to **both** the Android and iOS Wikipedia apps.
+
+## What Applib is not for
+* Android or iOS **specific** JS or CSS.
+* CSS unrelated to a particular Javascript transform. *In the future we may re-evaluate this for CSS common between the Android and iOS apps, but for right now the only CSS in Applib should be CSS directly needed by a particular Javascript transform.*
+
+## What Applib delivers
+* **applib.js** bundle of all transform JS
+* **applib.css** bundle of all CSS required by the bundled transform JS
```
Comment (niedzielski) [inline readme.md]: Noice!

File: readme.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+
+## Conventions
+
+### File locations and naming
+
+Example file names and locations for an image widening transform:
+* **src/WidenImage.js** - the transform. *required*
+* **src/WidenImage.css** - CSS used by the transform. *optional*
+* **test/WidenImage.js** - tests of the transform. *required*
```
Comment (niedzielski) [inline readme.md]: Consider calling out your test fixture too if it makes sense.

File: readme.md (hunk 1 above)
Comment (montehurd) [inline readme.md]: Fixed!

Comment (montehurd) [inline readme.md]: Fixed!

File: readme.md (hunk 3 above)
Comment (montehurd) [inline readme.md]: Fixed!

**Pull request merged**